### PR TITLE
Add typings for collection ids, deprecate groupId in some places

### PR DESCRIPTION
**Pull request by silberjan**

### CODE REVIEW

File: src/types/options/qualityRequestOptions.ts [hunk 1]
```diff
@@ -222,7 +224,9 @@ export interface MathCriterion extends BaseCriterion {
 interface BaseEdgeStatisticsCriterion extends BaseCriterion {
   type: 'edgeStatistics'
   edgeStatisticsServiceUrl?: string
-  edgeStatisticGroupId: number
+  /** @deprecated use edgeStatisticCollectionId id instead */
+  edgeStatisticGroupId?: number
+  edgeStatisticCollectionId?: number
   edgeStatisticId: number
```
Comment (HLucasTargomo) [inline src/types/options/qualityRequestOptions.ts:230]: edgeStatisticIds: number[]

File: src/types/options/qualityRequestOptions.ts [hunk 2]
```diff
@@ -222,7 +224,9 @@ export interface MathCriterion extends BaseCriterion {
 interface BaseEdgeStatisticsCriterion extends BaseCriterion {
   type: 'edgeStatistics'
   edgeStatisticsServiceUrl?: string
-  edgeStatisticGroupId: number
+  /** @deprecated use edgeStatisticCollectionId id instead */
+  edgeStatisticGroupId?: number
```
Comment (HLucasTargomo) [inline src/types/options/qualityRequestOptions.ts:228]: can you already remove this `edgeStatisticGroupId` from your library? so we can remove it soon from the service

File: src/types/options/statisticsRequestOptions.ts [hunk 3]
```diff
@@ -28,7 +29,8 @@ export interface StatisticsRequestOptionsSources extends StatisticsRequestOption
 
 export interface StatisticsGeometryRequestOptions {
   requestTimeout?: number
-  statisticsGroup: StatisticsGroupId
+  statisticCollectionId?: number
+  statisticsGroup?: StatisticsGroupId
```
Comment (HLucasTargomo) [inline src/types/options/statisticsRequestOptions.ts:33]: statisticsGroupId ?

File: src/types/options/statisticsRequestOptions.ts [hunk 4]
```diff
@@ -3,7 +3,8 @@ import { TravelRequestOptions } from '../requestOptions'
 import { GeometryId, LatLngId, TravelType } from '../types'
 
 export interface StatisticsTravelRequestOptions extends TravelRequestOptions {
-  statisticsGroup: StatisticsGroupId
+  statisticsGroup?: StatisticsGroupId
```
Comment (HLucasTargomo) [inline src/types/options/statisticsRequestOptions.ts:6]: statisticsGroupId ?

File: src/types/options/statisticsRequestOptions.ts (hunk 3 above)
Comment (silberjan) [reply src/types/options/statisticsRequestOptions.ts:33]: no its correct. was always called like that

File: src/types/options/qualityRequestOptions.ts (hunk 2 above)
Comment (silberjan) [reply src/types/options/qualityRequestOptions.ts]: done

File: src/types/options/qualityRequestOptions.ts (hunk 1 above)
Comment (silberjan) [reply src/types/options/qualityRequestOptions.ts:228]: nope

**Pull request merged**